feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/presenatation/account/listing/items/AccountUi.kt

```diff
@@ -10,5 +10,7 @@ class AccountUi(
     val isClickable: Boolean,
     val picture: Drawable,
     val chainIconUrl: String?,
+    val enabled: Boolean,
+    val updateIndicator: Boolean,
```

Comment:
I suggest to rename it to 'showUpdateInticator` or something similar to match usual Boolean namings